### PR TITLE
Improve workshop pages navigation

### PR DESCRIPTION
**Pull request by krzys-h**
Workshop pages navigation is now unified to have links to proposal, public page and qualification on all pages rather than a random structure

### CODE REVIEW

File: templates/_workshopeditheader.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% if workshop.status == 'Z' %}
```
Comment (grzegorz225) [inline templates/_workshopeditheader.html]: Use Enums instead of magic constants.

Comment (krzys-h) [reply templates/_workshopeditheader.html]: This is how it works everywhere in the project - add it to 'refactor all the things' list

Comment (grzegorz225) [reply templates/_workshopeditheader.html]: Ok

File: templates/workshop.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
         <h1>{{ title }}</h1>
         {% if has_perm_to_edit %}
             {% if workshop.status == 'Z' %}
-                <a href="{% url 'workshop_page' workshop.name%}">Edycja strony Twoich warsztatów</a>
+                {% include '_workshopeditheader.html' %}
```
Comment (cytadela8) [inline templates/workshop.html]: move this common part before if

File: templates/workshop.html [hunk 3]
```diff
@@ -21,7 +21,7 @@ <h1>{{ title }}</h1>
             </div>
         {% else %}
             {% if workshop.status == 'Z' %}
-                <a href="{% url 'workshop_page' workshop.name%}">Link do głównej strony warsztatów</a>
+                {% include '_workshopeditheader.html' %}
```
Comment (cytadela8) [inline templates/workshop.html]: this ^

**Pull request merged**